src/lib/components/Stepper.react.js

```diff
@@ -92,6 +92,11 @@ export default class Stepper extends Component<Props, State> {
     };
   }
 
+  UNSAFE_componentWillReceiveProps(nextProps: Props, nextContext: *): void {
```

Comment:
Why is this unsafe?

Reply:
legacy methods deprecated by React itself https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops